Orange/widgets/data/owfile.py

```diff
@@ -154,6 +154,7 @@ class OWFile(widget.OWWidget):
         doc="Attribute-valued data set read from the input file.")]
 
     want_main_area = False
+    resizing_enabled = False
```

Comment:
Said PR also includes a change that stretches the combo's drop-down listbox to accommodate its contents, regardless of combo's size.